tests/e2e/virt_backup_restore_suite_test.go

```diff
@@ -200,6 +200,8 @@ var _ = ginkgo.Describe("VM backup and restore tests", ginkgo.Ordered, func() {
 		gomega.Expect(err).To(gomega.BeNil())
 		err = v.CreateWaitForFirstConsumerStorageClass("test-sc-wffc")
 		gomega.Expect(err).To(gomega.BeNil())
+		lib.DeleteBackupRepositoryByRegex(runTimeClientForSuiteRun, "openshift-oadp", "cirros-test.*")
```

Comment:
I think this namespace should be "openshift-adp" instead.

Reply:
There is a variable somewhere in test suite for namespace IIRC.

Reply:
oh ur right, I had found that and I guess I didn't get it committed.  I'll update

Reply:
updated to use global, thank you!